---
/**:
  ros__parameters:
    timeout: 5.0
    temp_cold_warn: -15.0
    temp_cold_error: -20.0
    temp_hot_warn: 75.0
    temp_hot_error: 80.0
    rpm_ratio_warn: 0.80
    rpm_ratio_error: 0.70
